ui/components/app/app-header/app-header.component.js

```diff
@@ -115,30 +116,38 @@ export default class AppHeader extends PureComponent {
     } = this.props;
 
     return (
-      <div className="app-header">
-        <div className="app-header__contents">
-          <MetaFoxLogo
-            unsetIconHeight
-            onClick={async () => {
-              if (onClick) {
-                await onClick();
-              }
-              history.push(DEFAULT_ROUTE);
-            }}
-          />
-          <div className="app-header__account-menu-container">
-            {!hideNetworkIndicator && (
-              <div className="app-header__network-component-wrapper">
-                <NetworkDisplay
-                  onClick={(event) => this.handleNetworkIndicatorClick(event)}
-                  disabled={disabled || disableNetworkIndicator}
-                />
-              </div>
-            )}
-            {this.renderAccountMenu()}
+      <>
+        {
+          ///: BEGIN:ONLY_INCLUDE_IN(beta)
+          <BetaHomeHeader />
```

Comment:
a test case in app-header.test.js checking that this is only shown if isBeta key is set would be nice.